cilium/cmd/config.go

```diff
@@ -77,12 +78,26 @@ func configDaemon(cmd *cobra.Command, opts []string) {
 		dumpConfig(resp.Configuration.Mutable)
 		fmt.Printf("%-24s %s\n", "k8s-configuration", resp.K8sConfiguration)
 		fmt.Printf("%-24s %s\n", "k8s-endpoint", resp.K8sEndpoint)
+		fmt.Printf("%-24s %s\n", "PolicyEnforcement", resp.PolicyEnforcement)
```

Comment:
I'll see if I can remove the old Policy boolean option entirely from the daemon. I only think it is used in the daemon in `daemon/daemon.go:writeNetdevHeader()`. I can create a function that writes the `#define` for PolicyEnforcement for the daemon.